### PR TITLE
GEN2 | Predictions | guardian feedback

### PR DESCRIPTION
**Pull request by offlineprogrammer**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/[platform]/build-a-backend/add-aws-services/predictions/set-up-predictions/index.mdx [hunk 1]
```diff
@@ -86,12 +86,15 @@ const backend = defineBackend({
   auth,
 });
 
+
+// Configure a policy for the required use case.
+// The actions included below cover all supported ML capabilities
```
Comment (salihgueler) [inline src/pages/[platform]/build-a-backend/add-aws-services/predictions/set-up-predictions/index.mdx:91]: I think this is too hidden, I think we should use a Callout and direct customers to the policy documentations. 

e.g.

"With the following we are adding the policies for all the usecases in this documentation. There is a high chance that you won't need all of these policies. Check the docs .... to select the correct policy"

**Review by salihgueler — CHANGES_REQUESTED** [review]

File: src/pages/[platform]/build-a-backend/add-aws-services/predictions/set-up-predictions/index.mdx (hunk 1 above)
Comment (ykethan) [reply src/pages/[platform]/build-a-backend/add-aws-services/predictions/set-up-predictions/index.mdx:91]: was wondering if we could either add a comment before each group of policies with something like `permissions to enable translate capabilities` or split the policies to their individual docs for readability

Comment (salihgueler) [reply src/pages/[platform]/build-a-backend/add-aws-services/predictions/set-up-predictions/index.mdx:91]: I am okay with whatever way can teach customers about the policies and limit the usage of it to the only ones that they need.

Comment (offlineprogrammer) [reply src/pages/[platform]/build-a-backend/add-aws-services/predictions/set-up-predictions/index.mdx:98]: Updated to add a callout, I kept the comment inside the code from more visibility, didn't spilt it up for readability

**Review by ykethan — DISMISSED** [review]

**Review by ykethan — APPROVED** [review]

**Review by salihgueler — APPROVED** [review]
Looks good, thanks for the changes.

**Review by timngyn — APPROVED** [review]

**Pull request merged**